synapse/tests/test_lib_stormtypes.py

```diff
@@ -5036,15 +5011,16 @@ async def test_storm_lib_lift(self):
             nodes = await core.nodes('yield $lib.lift.byNodeData(newp) $node.data.load(lulz)')
             self.len(0, nodes)
 
-            nodes = await core.nodes('yield $lib.lift.byNodeData(hehe) $node.data.load(lulz)')
-            self.len(1, nodes)
-            self.eq(('inet:ipv4', 0x01020304), nodes[0].ndef)
-            self.eq('haha', nodes[0].nodedata['hehe'])
-            self.eq('haha', nodes[0].pack()[1]['nodedata']['hehe'])
-            self.eq('rofl', nodes[0].nodedata['lulz'])
-            self.eq('rofl', nodes[0].pack()[1]['nodedata']['lulz'])
+            msgs = await core.stormlist('yield $lib.lift.byNodeData(hehe) $node.data.load(lulz)')
```

Comment:
this was the nodedata change; correct?

Reply:
Yeah, loading the data is no longer done automatically since it isn't included in the sode when the layer is lifting it, it now has to be loaded into the path after lifting.